core/src/main/res/drawable/ic_download.xml

```diff
@@ -1,5 +1,10 @@
-<vector android:height="24dp"
-    android:viewportHeight="24.0" android:viewportWidth="24.0"
-    android:width="24dp" xmlns:android="http://schemas.android.com/apk/res/android">
-    <path android:fillColor="?attr/action_icon_color" android:pathData="M19,9h-4V3H9v6H5l7,7 7,-7zM5,18v2h14v-2H5z"/>
-</vector>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24"
+    android:tint="?attr/action_icon_color">
+    <path
+        android:fillColor="@android:color/white"
```

Comment:
Instead of a tinted white path, please use the color directly.

```suggestion
    android:viewportHeight="24">
    <path
        android:fillColor="?attr/action_icon_color"
```

Reply:
Thanks for the suggestion.